src/Makefile.am

```diff
@@ -3,6 +3,8 @@ noinst_PROGRAMS = sandbox gen-sum
 
 noinst_LIBRARIES = libsandbox.a
 
+AM_CXXFLAGS = -std=c++11
```

Comment:
Is c++11 required?  We have avoided this in the main bitcoin codebase, and my default preference is to be as liberal WRT old compilers as possible.